delta/plugins/archive/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/ArchiveDownload.scala

```diff
@@ -132,11 +161,12 @@ object ArchiveDownload {
             case FileRejection.AuthorizationFailed(addr, perm) => AuthorizationFailed(addr, perm)
             case other                                         => WrappedFileRejection(other)
           }
-          .flatMap { fileResponse =>
+          .flatMap { case FileResponse(fileMetadata, content) =>
             IO.fromEither(
-              pathOf(ref, project, format, fileResponse.filename).map { path =>
-                val metadata = format.metadata(path, fileResponse.bytes)
-                Some((metadata, fileResponse.content))
+              pathOf(ref, project, format, fileMetadata.filename).map { path =>
+                val archiveMetadata               = format.metadata(path, fileMetadata.bytes)
+                val contentTask: Task[AkkaSource] = content.mapError(_ => new RuntimeException())
```

Comment:
Better to create an `ArchiveDownloadError` that extends `SDKError` than the runtime one
+ `.tapError` to log the error